packages/teleport-cli/src/services/index.ts

```diff
@@ -0,0 +1,123 @@
+import path from 'path'
+import fetch from 'node-fetch'
+import { ensureDirSync } from 'fs-extra'
+import {
+  CompiledComponent,
+  ComponentType,
+  ComponentUIDL,
+  ProjectType,
+  ProjectUIDL,
+  VComponentUIDL,
+  VProjectUIDL,
+  GeneratedFolder,
+  FileType,
+} from '@teleporthq/teleport-types'
+import { generateComponent, packProject } from '@teleporthq/teleport-code-generator'
+import { injectFilesFromSubFolder, injectFilesToPath, findFileByName } from './file'
+import {
+  BASE_URL,
+  STUDIO_URL,
+  UUDID_REGEX,
+  CONFIG_FILE_NAME,
+  DEFALT_CONFIG_TEMPLATE,
+  DefaultConfigTemplate,
+  DEFAULT_CONFIG_FILE_NAME,
+} from '../constants'
+import { getPackageJSON } from '../utils'
+
+export const fetchUIDLFromREPL = async (url: string): Promise<Record<string, unknown>> => {
+  const id = url.match(UUDID_REGEX)[0]
```

Comment:
I think here is better to use this https://www.npmjs.com/package/uuid#uuidvalidatestr